wagtail_transfer/views.py

```diff
@@ -190,6 +239,57 @@ def do_import(request):
         return redirect('wagtailadmin_explore_root')
 
 
+def import_model(request):
+    source = request.POST['source']
+    model = request.POST['source_model']
+    base_url = settings.WAGTAILTRANSFER_SOURCES[source]['BASE_URL']
+    digest = digest_for_source(source, model)
+
+    url = f"{base_url}api/models/{model}/"
+    if request.POST.get("source_model_object_id"):
+        source_model_object_id = request.POST.get("source_model_object_id")
+        url = f"{url}{source_model_object_id}/"
+
+    response = requests.get(url, params={'digest': digest})
+    importer = ImportPlanner(model=model)
+    importer.add_json(response.content)
+
+    while importer.missing_object_data:
+        # convert missing_object_data from a set of (model_class, id) tuples
+        # into a dict of {model_class_label: [list_of_ids]}
+        missing_object_data_by_type = defaultdict(list)
+        for model_class, source_id in importer.missing_object_data:
+            missing_object_data_by_type[model_class].append(source_id)
+
+        request_data = json.dumps({
+            model_class._meta.label_lower: ids
+            for model_class, ids in missing_object_data_by_type.items()
+        })
+        digest = digest_for_source(source, request_data)
+
+        # request the missing object data and add to the import plan
+        response = requests.post(
+            f"{base_url}api/objects/", params={'digest': digest}, data=request_data
+        )
+        importer.add_json(response.content)
+    importer.run()
+
+    messages.add_message(request, messages.SUCCESS, 'Snippet(s) successfully imported')
+    app_label, model_name = model.split('.')
+    redirect_url = reverse('wagtailsnippets:list', args=(app_label, model_name,))
```

Comment:
I think you can write this as: ``return redirect('wagtailsnippets:list', app_label, model_name)``